python/pypto/language/op/tile_ops.py

```
@@ -2039,6 +2043,36 @@ def part_min(src0: Tile, src1: Tile) -> Tile:
+def axpy(src: Tile, scalar: int | float | Expr | Scalar, dst: Tile) -> Tile:
+    """Accumulate ``dst += src * scalar`` in the destination tile."""
+    scalar_expr = scalar.unwrap() if isinstance(scalar, Scalar) else scalar
+    return Tile(expr=_ir_ops.axpy(src.unwrap(), scalar_expr, dst.unwrap()))
+
+
+def add_relu(src0: Tile, src1: Tile) -> Tile:
+    """Fused element-wise add followed by ReLU."""
+    return Tile(expr=_ir_ops.add_relu(src0.unwrap(), src1.unwrap()))
+
+
+def pow(base: Tile, exp: Tile, tmp: Tile | None = None, *, high_precision: bool = False) -> Tile:
```

Ruff (0.16.0)

[error] Variable `pow` is shadowing a Python builtin

(A001)